blogs/network_events/index.md

```diff
@@ -0,0 +1,208 @@
+various network activities with minimal performance overhead, allowing you to observe and capture detailed
+information about network traffic and events in real-time.
+
+## Key Components for Network Observability with eBPF
+
+1- eBPF network events monitoring `Kprobe` eBPF hook
+
+Network events monitoring using eBPF kernel probes (kprobes) provides deep, kernel-level insights into
+network stack behavior. The NetObserv eBPF agent leverages the [psample_sample_packet](https://github.com/torvalds/linux/blob/e32cde8d2bd7d251a8f9b434143977ddf13dcec6/net/psample/psample.c#L363) entry point to efficiently
+capture packet metadata and identify policy violations with minimal overhead.
+
+The hook implemented in the NetObserv eBPF agent can capture multiple events within the same network flow
+and generate a list of network events that applied to that flow, with a limit of up to four events per flow.
+
+2- [ovn-kubernetes observability library](https://github.com/ovn-org/ovn-kubernetes/tree/master/go-controller/observability-lib)
+
+The eBPF agent captures network events as an array of bytes, which is not very user-friendly.
+This library provides functionality to convert these events into human-readable strings, making
+them easily understandable for customers.
+
+## Use Cases for Network Observability with eBPF
+
+1. Monitor OVN network policies
+
+When you apply OVN network policies (like Kubernetes NetworkPolicy), eBPF agent can monitor allowed
+and/or blocked traffic, detecting whether packets are allowed or blocked based on network policies.
+An example of these network policies follows:
+
+```yaml
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: iperf3-server-access-ingress
+  namespace: iperf
+spec:
+  podSelector:
+    matchLabels:
+      app: iperf3-server
+  ingress:
+  - ports:
+    - port: 5501
+    from:
+    - podSelector:
+        matchLabels:
+          app: iperf3-client
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: iperf3-server-access-egress
+  namespace: iperf
+spec:
+  podSelector:
+    matchLabels:
+      app: iperf3-client
+  egress:
+  - ports:
+    - port: 5201
+      protocol: TCP
+  podSelector: {}
+  policyTypes:
+  - Egress
+```
+
+![ingress and egress network policy events](./images/ingr_egr_network_events.png)
+_Screen capture of the traffic flows in the NetObserv console plugin,
+with a network events column showing traffic allowed or denied by network policy rules_
+
+2. Admin network Policies
+
+An Admin network Policy is a high-level security policy designed to provide cluster administrators with
+more control over network traffic in Kubernetes environments.
+eBPF agent can monitor allowed/blocked Traffic: Detect whether packets are allowed or blocked based on
+admin network policies.
+
+An Example of admin network policy and egress network policy to show multiple events use case :
```

Comment:
do you think including the egress netpol here would be helpful? maybe as a link to the previous section at least
otherwise may seem like this is the effect of ANP alone